app/assets/javascripts/editor.js

```diff
@@ -474,7 +485,7 @@ function (ko, models, tools, msg, validate, owned, astate) {
                 self.editedAbstract(self.abstract());
 
                 var hasNoFig = !self.hasAbstractFigures(),
-                    hasFigData = self.newFigure.file ? true : false;
+                    hasFigData = !!self.newFigure.file;
```

Comment:
Please move this change to its own commit. It would not be findable when searching for it through the commit message.